.circleci/config.yml

```diff
@@ -31,11 +96,50 @@ jobs:
             ssh -o StrictHostKeyChecking=no "${PRODUCTION_BACKUP_LOCATION}" -T >/dev/null
             $DEVOPS_WORKDIR/rethinkdb/rethinkdb-backup.sh \
               -s "${PRODUCTION_BACKUP_LOCATION}" -d "${PRODUCTION_BACKUP_VOLUME}" \
-              -b "${S3_DB_BACKUPS_BUCKET}" -p "${CIRCLE_BRANCH}"
+              -b "${S3_DB_BACKUPS_BUCKET}" -p production
       - run:
           name: Slack completion
           command: |
             curl --ssl -X POST -H 'Content-type: application/json' --data '{"text":"`production` Backup done"}' $SLACK_DEVOPS_URL
+  staging_rethinkdb_restore:
+    docker:
+      - image: cimg/node:18.13.0
+    environment:
+      DEVOPS_REPO: "git@github.com:ParabolInc/action-devops.git"
+      DEVOPS_WORKDIR: "~/action-devops"
+      STAGING_BACKUP_LOCATION: "backups.action-staging.parabol.co"
+      S3_DB_BACKUPS_BUCKET: "db-backups.parabol.co"
+    working_directory: ~/action
+    resource_class: medium
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "53:a8:37:35:c3:7e:54:f5:19:f6:8e:a1:e0:78:52:da"
+      - run:
+          name: Slack setup
+          command: |
+            curl --ssl -X POST -H 'Content-type: application/json' --data '{"text":"Starting `Staging` RethinkDB wipe out and restore..."}' $SLACK_DEVOPS_URL
+      - checkout
+      - run:
+          name: DevOps checkout
+          command: |
+            git clone $DEVOPS_REPO $DEVOPS_WORKDIR
```

Comment:
-1 see above, this is cloned but then unused

Reply:
Fixed, but to be tested.